<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advert</class>
 <widget class="QDialog" name="Advert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelText">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Вітаю, ви виграли!</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelTime">
           <property name="text">
            <string>Час: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="insertTime">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="labelBestTime">
           <property name="text">
            <string>Кращий час: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="insertBestTime">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="labelPlayed">
           <property name="text">
            <string>Ігр зіграно: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="insertPlayed">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="labelWon">
           <property name="text">
            <string>Ігр виграно: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="insertWon">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="labelDate">
           <property name="text">
            <string>Дата: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="insertDate">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="labelPercentage">
           <property name="text">
            <string>Відсоток: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="insertPercentage">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushExit">
       <property name="text">
        <string>Вийти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushRestart">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="text">
        <string>Повторити цю гру</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushAgain">
       <property name="text">
        <string>Грати знову</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushAgain</tabstop>
  <tabstop>pushRestart</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushExit</sender>
   <signal>clicked()</signal>
   <receiver>Advert</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAgain</sender>
   <signal>clicked()</signal>
   <receiver>Advert</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
